NitroxClient/MonoBehaviours/NitroxBootstrapper.cs

```diff
@@ -25,6 +25,8 @@ private void Awake()
         // This is very important, see Application_runInBackground_Patch.cs
         Application.runInBackground = true;
         Log.Info($"Unity run in background set to \"{Application.runInBackground}\"");
+        // Also very important for similar reasons
```

Comment:
Would the similar reasons be in the same patch? If not much better good to specify where they can be found

Reply:
There's an XML comment in the said patch which explains the issue

Reply:
I assumed it was referencing that one just wanted to make sure as it wasn't 100% clear